src/Microsoft.OpenApi.Readers/YamlConverter.cs

```diff
@@ -0,0 +1,133 @@
+﻿using System;
+using System.Collections.Generic;
+using System.Linq;
+using System.Text.Json.Nodes;
+using Json.More;
+using SharpYaml;
+using SharpYaml.Serialization;
+
+namespace Microsoft.OpenApi.Readers
+{
+    /// <summary>
+    /// Provides extensions to convert YAML models to JSON models.
```

Comment:
This is awesome.  Thanks!  Maggie is currently doing a spike to try and convert the ParseNode classes to work off JsonNode instead of YamlNode.  If she is able to do that, then this class will enable us to continue supporting YAML parsing.

Reply:
This code is basically copied from my YamlToJsonNode package.  That lib uses YamlDotNet which is the basis for YamlSharp, so it's the same code for "different" types.